config/src/main/java/org/springframework/security/config/annotation/web/configurers/RememberMeConfigurer.java

```diff
@@ -186,6 +189,24 @@ public RememberMeConfigurer<H> key(String key) {
 		return this;
 	}
 
+	/**
+	 * The algorithm to use with {@link TokenBasedRememberMeServices} for hashing the
+	 * digital signature containing the key from {@link #key(String)} and the user's
+	 * password. If unset, the MD5 message digest algorithm will be used.
+	 * <p>
+	 * This configuration is ignored if
+	 * {@link #tokenRepository(PersistentTokenRepository)} or
+	 * {@link #rememberMeServices(RememberMeServices)} are used.
+	 * @param hashingAlgorithm the algorithm used when creating new cookies
+	 * @return the {@link RememberMeConfigurer} for further customization
+	 * @since 5.7
+	 */
+	public RememberMeConfigurer<H> hashingAlgorithm(RememberMeHashingAlgorithm hashingAlgorithm) {
```

Comment:
Since `RememberMeServices` can be constructed with relative ease, I'd recommend leaving this out of the DSL for the time being.